example/pubspec.lock

```diff
@@ -355,7 +355,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.5.2"
+    version: "1.9.0"
```

Comment:
Whoah, what happened here? 😄 Is this a runnable example that had been pointing to an old version of the library?

Reply:
yes